astropy/modeling/functional_models.py

```diff
@@ -3549,12 +3550,13 @@ def inverse(self):
         new_tau = self.amplitude
         return Logarithmic1D(amplitude=new_amplitude, tau=new_tau)
 
-    @tau.validator
-    def tau(self, val):
```

Comment:
Since you want to backport, is this fully backward compatible?

Reply:
These validator functions are not part of the public API. I don't believe anyone is using them for any reason. So I'd say this is backwards compatible.